.github/workflows/e2e-tests.yml

```diff
@@ -144,6 +144,13 @@ jobs:
         name: test-output-${{ matrix.group }}
         path: projects/plugins/jetpack/tests/e2e/output
 
+    - name: Send Slack notification
```

Comment:
I'm curious to know why you moved the step here.

Reply:
I didn't want a failure in the `testspace` step to trigger a test failure notification.